src/cpu/aarch64/injectors/jit_uni_eltwise_injector.cpp

```diff
@@ -0,0 +1,2253 @@
+/*******************************************************************************
+* Copyright 2019-2020 Intel Corporation
+* Copyright 2020 FUJITSU LIMITED
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "common/c_types_map.hpp"
+#include "common/dnnl_thread.hpp"
+#include "common/nstl.hpp"
+#include "common/utils.hpp"
+
+#include "cpu/aarch64/injectors/jit_uni_eltwise_injector.hpp"
+
+#define IDX(a) static_cast<uint32_t>(a.getIdx())
+
+namespace dnnl {
+namespace impl {
+namespace cpu {
+namespace aarch64 {
+
+using namespace Xbyak_aarch64;
+
+template <cpu_isa_t isa>
+void jit_uni_eltwise_injector_f32<isa>::injector_preamble(
+        const injector_utils::vmm_index_set_t &vmm_idxs) {
+    using namespace alg_kind;
+    using namespace Xbyak_aarch64::util;
+    preserved_vecs_count = 0;
+    vecs_to_preserve = aux_vecs_count();
+    const auto start_idx = *(vmm_idxs.begin());
+    const auto end_idx = *(vmm_idxs.rbegin()) + 1;
+    start_idx_tail = vmm_idxs.begin();
+
+    // For sse41 mask register has to be Xmm(0)
```

Comment:
does this restriction applies to aarch64?
In any case, please modify the comment to not mention sse and Xmms on aarch64.

Reply:
At the moment, aarch64 eltwise supports only sve_512, and these lines are not required.
I'll delete it.